content/en/docs/Installation/pre-built-binaries.md

```diff
@@ -9,6 +9,12 @@ description: >
 Just head to the [releases page](https://github.com/deluan/navidrome/releases) and download the latest version for you 
 platform. There are builds available for Linux (amd64 and arm), Windows (32 and 64 bits) and macOS.
 
+For [Arch Linux](https://archlinux.org), you can install Navidrome via the
+[`navidrome-bin`](https://aur.archlinux.org/packages/navidrome-bin/) package in
```

Comment:
I think we should add a notice explaining that the `navidrome-bin` may be outdated. Is there a way to query AUR repo to see which version is there?

Reply:
Maybe we could put AUR instructions in a separated paragraph, with a proper header
(something like `### Arch Linux Packages`)